CURRICULUM VITAE
---
Nov 2016

---
#Yingxiang LI
PhD candidate (will graduate in Jan 2017)  
Major in bioinformatics  
School of Life Science and Technology, Tongji University  
Room 1012, Medicine Builiding  
1239 Siping Road, Shanghai, 200092, P.R. China.   
Tel:            13548672401        
Skype:          Michael.X.Li        
Email:          xlccalyx@qq.com  
Home:           [www.calyx.biz](http://www.calyx.biz/)  
Linkedin:       [www.linkedin.com/in/yingxiangli](https://www.linkedin.com/in/yingxiangli)  
Google Scholar: [scholar.google.com/citations?user=ex6VfOQAAAAJ&hl=en](http://scholar.google.com/citations?user=ex6VfOQAAAAJ&hl=en) (93 cited)  
Github:		  [github.com/Yamol](https://github.com/Yamol)
	

---
##EDUCATION

2005 - 2008  
**The First Middle School of Changsha**, Changsha, Hunan, P.R. China.  
High school student in the key class of science.  

2008 - 2012  
**Tongji University**, Shanghai, P.R. China.  
Bachelor’s Degree in Bioinformatics.  

2012 - 2013  
**Tongji University**, Shanghai, P.R. China.  
PhD candidate in Bioinformatics.  
Advisor: `Cheng Li`.  
Related Advisors: `Xiaole Shirley Liu`, `Yong Zhang`.  

2013 - 2017  
**Tongji University**, Shanghai, P.R. China.  
PhD candidate in Bioinformatics.  
Advisor: `Zhiping Weng`.  
Related Advisors: `Wen Xue`, `Cheng Li`, `Xiaole Shirley Liu`, `Yong Zhang`.  

2015 – 2016  
**University of Massachusetts Medical School**, Worcester, MA, United States.  
Joint-PhD under China Scholarship Council.  
Advisor: `Wen Xue`, `Zhiping Weng`.  
Related Advisors: `Guangping Gao`, `Daniel G. Anderson`.  

*:  
`Cheng Li`:	Principal investigator. School of Life Sciences, Peking University. PRC.  
`Xiaole Shirley Liu`:	Professor. Biostatistics and Computational Biology, Dana-Farber and Harvard. US.  
`Yong Zhang`:	Professor. School of Life Science, Tongji University. PRC.  
`Zhiping Weng`:	Director, Professor. Program in Bioinformatics and Integrative Biology, University of Massachusetts Medical School. US.  
`Wen Xue`:	Assistant Professor. Program in Molecular Medicine, University of Massachusetts Medical School. US.  
`Guangping Gao`:	Professor. Microbiology and Physiological Systems, University of Massachusetts Medical School. US.  
`Daniel G Anderson`:	Associate Professor, Chemical Engineering and Institute for Medical Engineering and Science, Massachusetts Institute of Technology. US.  

---
##PUBLICATIONS
(#: co-first author)

1.	_Fulciniti, M., Li, Y., … Munshi, N. (2012) Integrating Gene and Mir Expression Profiles and Regulatory Network Structures to Define Aberrent Feed Forward Loops with Functional and Clinical Implications in Myeloma. **BLOOD** (Conference Poster)._   
IF: 10.5 [_https://ash.confex.com/ash/2012/webprogram/Paper54063.html_](https://ash.confex.com/ash/2012/webprogram/Paper54063.html "")  
Published in November 2012 in BLOOD annual meeting of AMER SOC HEMATOLOGY. I helped the cooperator on data collection and analysis, which focused on regular network structure and feed forward loops of multiple myeloma.

2.	_Li, Y.#, … Li, C. (2013). Classify hyperdiploidy status of multiple myeloma patients using gene expression profiles. **PloS One**, 8(3), e58809._   
IF: 3.2 [_http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0058809_](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0058809 "")  
Published in March 2013 during my first year as a PhD candidate. We designed and realized the classification of multiple myeloma into hyperdiploidy and non-hyperdiploidy status by gene expression profiling.

3.	_Wang, X., Yan, Z., Fulciniti, M., Li, Y., … Li, C. (2014). Transcription factor-pathway coexpression analysis reveals cooperation between SP1 and ESR1 on dysregulating cell cycle arrest in non-hyperdiploid multiple myeloma. **Leukemia**, 28(4), 894–903._   
IF: 10.4 [_http://www.nature.com/leu/journal/v28/n4/full/leu2013233a.html_](http://www.nature.com/leu/journal/v28/n4/full/leu2013233a.html)  
This work was published in Augest 2013 after I helped my elder mate with his topic. It’s about co-expression analysis that effect transcription factor pathway in multiple myeloma and focus on SP1/ESR1.

4.	_Li, Y.#, … Xue, W. (2015). A versatile reporter system for CRISPR-mediated chromosomal rearrangements. **Genome Biology**, 16(1)._  
IF: 10.8 [_http://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0680-7_](http://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0680-7)  
This work was published in May 2015. It was the first co-operation with the CRISPR/Cas9 experiment lab. I designed part of the study and worked on the bioinformatics section.

5.	_Wang, D., Mou, H., Li, S., Li, Y., … Xue, W. (2015). Adenovirus-mediated somatic genome editing of Pten by CRISPR/Cas9 in mouse liver in spite of Cas9-specific immune responses. **Human Gene Therapy**, 26(7), 432–442._   
IF: 3.8 [_http://online.liebertpub.com/doi/abs/10.1089/hum.2015.087_](http://online.liebertpub.com/doi/abs/10.1089/hum.2015.087)  
This work was published in July 2015. I mainly worked on the bioinformatics part to calculate the indel frequency created by CRISPR/Cas9.

6.	_Yin, H., Song, C.-Q., Dorkin, J.R., Zhu, L.J., Li, Y., …Anderson, G.D. (2016). Therapeutic genome editing by combined viral and non-viral delivery of CRISPR system components in vivo. **Nat. Biotechnol.** 34, 328–333._  
IF: 41.5 [_http://www.nature.com/nbt/journal/v34/n3/full/nbt.3471.html_](http://www.nature.com/nbt/journal/v34/n3/full/nbt.3471.html)  
This work was published in Feb 2016. I mainly supported the bioinformatic analysis and plot the indel frequency created by CRISPR.

7.	2 more papers are under submission.

---
##RESEARCH EXPERIENCE

* 2009 - 2011  
**The character and biological effect of transmission module of gene relationship**  
It was Students Innovation Training Program in Tongji University. This project was about an investigation of genes in one orthologous group. We measured the linear distance between genes from basic organisms to complex ones. This program was supervised by `Haiyun Wang` (Tongji University).

* 2011 - 2012  
**Integrating gene and Mir expression profiles and regulatory network structures to define aberrant feed forward loops with functional and clinical implications in myeloma**  
We have here developed and defined multi-gene transcriptional and post-transcriptional feed-forward loop (FFL). These conceptual FFLs consist of a master TF which regulates a miR and together with it controls a set of specific common gene/s. We have developed a comprehensive novel integrative analysis method, dChip-GemiNI (Gene and miRNA Network-based Integration), which combines gene and miR expression profiles, and also incorporates regulatory network structure in the form of computationally identified TF–miRNA FFLs. This work was exhibited as conference poster in 54th ASH Annual Meeting and Exposition held by _**Blood**_. This program was supervised by `Cheng Li` and `Nikhil C. Munshi` (DFCI, Harvard Medical School).

* 2011 - 2013   
**Classify hyperdiploidy status of multiple myeloma patients using gene expression profiles**  
Multiple myeloma (MM) frequently harbors alterations in DNA and chromosome copy numbers, and can be divided into two major subtypes, hyperdiploid (HMM) and non-hyperdiploid multiple myeloma (NHMM). The two subtypes have different survival prognosis, possibly due to different but converging paths to oncogenesis. Using five MM expression datasets that have HMM status measured by FISH and copy number microarrays, we have developed and validated a K-nearest-neighbor method to classify MM into HMM and NHMM based on gene expression profiles. We published our result in _**PloS One**_. This program was supervised by `Cheng Li`.

* 2012- 2014	 
**Transcription factor-pathway coexpression analysis reveals cooperation between SP1 and ESR1 on dysregulating cell cycle arrest in non-hyperdiploid multiple myeloma**  
Transcription factors (TrFs) have been implicated in myeloma oncogenesis, but their dysregulation in myeloma subtypes are less studied. Here, we developed a TrF-pathway coexpression analysis to identify altered coexpression between two sample types. We find that TrFs MYC, nuclear factor-kB and HOXA9 have significantly lower coexpression with cell cycle arrest in HMM, co-occurring with their overactivation in HMM. In contrast, TrFs ESR1 (estrogen receptor 1), SP1 and E2F1 have significantly lower coexpression with cell cycle arrest in NHMM. This work was published in Leukemia. This program was supervised by `Cheng Li`.

* 2013 – 2014	 
**Cell cycle genes and related miR expression analysis**  
This program is under the cooperation with Dana-Farber Cancer Institute and Harvard Medical School. We are analyzing the relation between miR and mRNA in cell cycle pathway. This program was supervised by `Cheng Li` and `Peter Sicinski` (Harvard University).

* 2013 – 2014	 
**Squamous cell lung carcinoma related analysis**  
This program is under the cooperation with Shanghai Chinese Academy of Sciences. We are analyzing the SCC’s pathogenesis and related gene expression/copy number/gene mutation. This program was supervised by `Cheng Li` and `Hongbin Ji` (Shanghai Institutes for Biological Sciences).

* 2013 – 2015	 
**Bi-state gene in cancer**  
We hypothesized that there’re genes that could show different subtypes of cancer which we called them bi-state genes. We wanted to find out the genes that have such potential, and they could be regarded as key genes for determining subtypes of cancers. We combined the TCGA (The Cancer Genome Atlas) data and statistic method. This program was supervised by `Zhiping Weng`.

* 2013 – 2015	 
**Role of X gene (trade secret of Novartis)**  
We are interested in one gene and we are researching its role in all kinds of regulation in cells. ChIP-Seq and ChIP-MS experiments have been performed on this gene. We have a deep cooperation with Novartis in Shanghai. This program was supervised by `Zhiping Weng` and `Bin Xiang` (Novartis International).

* 2014 – 2016 
**Studying protein P function in regulating protein B transcription activity (trade secret of Novartis)**  
We are under an ongoing drug discovery project. Multiple ChIP-Seq data are designed for search of their relationship. This project is also a cooperation with Novartis in Shanghai. It was supervised by `Zhiping Weng` and `Bin Xiang`.

* 2014 – 2015	 
**Identification of LncRNA in lung cancer**  
We collected lncRNA information of various types of cancers. At the same time, we gathered mutation data from different papers or public database. Then we hope to explore the mutation effect on lncRNA function only in the non-coding region, and try to find their influence on cancers. This program was supervised by `Zhiping Weng`.

* 2014 – 2015	 
**A versatile reporter system for CRISPR-mediated chromosomal rearrangements**  
We have developed a direct and quantifiable method that can measure CRISPR-mediated DNA inversion and deletion in cultured human and mouse cells as well as in mouse liver. Thus our CRISPR-based method can provide a platform for studying chromosomal rearrangements in cancers. Finally, we published our result in _**Genome Biology**_. This program was supervised by `Zhiping Weng` and `Wen Xue`.

* 2014 – 2015	 
**Integrated analysis of human hepatic carcinoma**  
We have made multiple experiments including ChIP-Seq, exome methylation, HBV integration sites, small RNA, transcriptome lincRNA, SNP and other high-through sequencing. All these data were integrated for a comprehensive analysis of human hepatic carcinoma. We hope to combine all these result to figure out a overall knowledge of liver cancer. This project is also an cooperation with Novartis in Shanghai. This program was supervised by `Zhiping Weng` and `Tianlun Zhou` (Novartis International).

* 2014 – 2015	 
**Adenovirus-mediated somatic genome editing of Pten by CRISPR/Cas9 in mouse liver in spite of Cas9-specific immune responses**  
We used adenovirus (Ad) vector to deliver a Streptococcus pyogenes-derived Cas9 system (SpCas9) targeting Pten, a gene involved in NASH and a negative regulator of the PI3K-AKT pathway, in mouse liver. Our findings provide a strategy to model human liver diseases in mice and highlight the importance considering Cas9-specific immune responses in future translational studies involving in vivo delivery of CRISPR/Cas9. Finally, we published our result in _**Human gene therapy**_.  This program was supervised by `Zhiping Weng`, `Wen Xue` and `Guangping Gao` (Massachusetts Institute of Technology).

* 2015 – 2016   
**Combined viral and nonviral delivery of CRISPR components corrects liver disease in adult rodents**  
The ability to correct defective genes in vivo has broad potential utility for both therapy and the study of genetics. By combining viral and nonviral nucleic acid delivery we report the first therapeutically relevant formulations capable of inducing repair of a disease gene in an adult animal. This treatment fully rescued body weight loss, alleviated liver damage-associated serum markers and generated Fah-positive hepatocytes. Our study indicates that systemic CRISPR-mediated genome editing is possible in vivo, and provides proof-of-principle for therapeutic correction of genetic diseases in adult animals. It was published in _**Nature biotechnology**_. This program was supervised by `Zhiping Weng`, `Wen Xue` and `Daniel G. Anderson` (Massachusetts Institute of Technology).

* 2015 – 2016	 
**Genome-wide CRISPR screen identifies MAPK regulators as key tumor suppressors in liver**  
Using a CRISPR-based genetic screen, we identify candidate liver tumor suppressors, including NF1-a negative regulator of Ras mutated in neurofibromatosis. CRISPR-mediated knockout of Nf1 accelerates liver tumor formation in mice, and loss of Nf1 or activation of Ras upregulates the liver progenitor cell markers Hmga2 and Sox9. In human liver cancer patients, low NF1 or high HMGA2 mRNA levels predict poor survival. Treatment of human liver cancer cells with RAS pathway inhibitors including sorafenib suppresses HMGA2 and SOX9 expression, and knockdown of Hmga2 delays tumorigenesis driven by oncogenic RAS. We are submitting the project to the journal. This program was supervised by `Zhiping Weng`, `Wen Xue`, supported by `China Scholar Council` (CSC).

* 2015 – 2016  
**CIpipe: A pipeline for analyzing insertions and deletions in DNA sequences caused by CRISPR**  
CRISPR-Cas9 is a powerful tool for sequence-specific genome editing. The Cas protein cuts genomic DNA at locations complementary to a single guide RNA. Insertions and deletions (indels) often result when the cuts are repaired. Currently, there is no easy-to-use computational pipeline to determine the locations, identities, and frequencies of the indels. We have developed a pipe-line, named CIpipe (CRISPR Indel pipeline), to identify indels in high-throughput DNA sequencing data and provide the statistical characterization of these indels. This program was supervised by `Zhiping Weng`, `Wen Xue`, supported by `China Scholar Council` (CSC). ([github.com/Yamol/CIpipe](github.com/Yamol/CIpipe))

* 2015 – 2016	 
**Genetic disruption of oncogenic Kras sensitizes lung cancer cells to Fas receptor-mediated apoptosis**  
Genetic lesions that activate KRAS account for ~30% of the 1.6 million annual cases of lung cancer. Despite clinical need, KRAS is still undruggable using traditional small molecule drugs/inhibitors. When oncogenic Kras is suppressed by RNA interference, tumors initially regress but eventually recur and proliferate despite suppression of Kras. Here we show that tumor cells can survive CRISPR-mediated knockout of oncogenic Kras, indicating the existence of Kras-independent survival pathways. We are submitting the project to the journal. This program was supervised by `Zhiping Weng`, `Wen Xue`, supported by `China Scholar Council` (CSC).

---
##HONORS AND AWARDS 

* 2009  
**Outstanding student scholarship** of Tongji University, second class  
* 2009  
The 20th Qingnian Cup inter-college **debate competition** of Tongji University, **second place** (a member of the college team)  
* 2010  
**Outstanding student** of Tongji University  
* 2010  
The Tongbo Cup **football match** of Tongji University, **first place** (a member of the college team)  
* 2011  
**Excellent league member** of Tongji University  
* 2011  
**Learning scholarship** of Tongji University, third class  
* 2013  
**National scholarship for graduate students**, master  
* 2015  
**Joint-PhD** supported by China Scholarship Council, one year  
* 2015  
**National scholarship for graduate students**, PhD  

---
##EXTRACURRICULAR EXPERIENCES

* 2008 - 2009	
The **monitor**. Process the routines of the class. School of life science and technology, Tongji University.
* 2009	
The **member** of college debate team and won the second place. 
* 2009 - 2010	
The **vice president** of college student union. Help the college with common routines. 
* 2010	
A **forward** in our college soccer team and finally won the champion. 
* 2010 - 2011	
The **president** of college student union. Manage the students’ routines of our college, and arranged a graduate ceremony as the director.
* 2011	
A **volunteer** in the Tongji-Novartis Cancer and Epigenetics Symposium. Help with the meeting affairs. 
* 2012	
A **teaching assistant**: Statistical Model by Prof. Li Cheng. 

---
##SOCIAL WORK EXPERIENCES

* 2009	
**A volunteer** of the Red Cross. Help to advertise the knowledge of AIDS and help the school hospital with medical affair. Tongji University.
* 2009	
**The team leader** of an internship about advertisement of the CBA. Help to investigate the market of sportswear. Shanghai.

---
##LANGUAGE PROFICIENCY

* **CET 6**	540.
* **TOEFL**	90.

---
##SKILLS

* bioinformatics, biology, cancer research, non-invasive prenatal diagnosis, genetics, Dota2, sequencing analysis, programme, big data, data mining, R, python, shell, Cytoscape, Office Suit and so on.
